### ISSUE TITLE
No themes registered after install on vanilla emacs

### ISSUE THREAD

**Issue by nrvale0** [ronisbr/doom-nano-themes#1]
Howdy.

I'd like to use these theme outside of Doom Emacs / in vanilla Emacs but after installing the theme package and the associated modeline package I'm not able to M-x load-theme them. The themes don't show up in the last of completions. Here's how I'm installing...

    (use-package doom-nano-modeline
      :straight (:host github :repo "ronisbr/doom-nano-modeline"))

    (use-package doom-nano-themes
      :straight (:hist github :repo "ronisbr/doom-nano-themes"))

I set toggle-debug-on-error but the debugger isn't firing up so I think there's not an error. 

Also, consistent with that thought, from \*Messages\*:

    Loading package doom-nano-modeline...done
    Loading package doom-nano-themes...done

which is a pretty strong indicator of a clean load.

Perhaps there's something that the doom-themes framework does automagically that needs to be done via some other method outside of Doom?

**Comment by ronisbr** [comment]
Hi @nrvale0 !

This theme is for Doom-themes. You need to install Doom themes and place the files in `~/.doom.d/themes`. As far as I know, you can install Doom-themes in vanilla emacs. Can you please try that?

The idea is to submit this theme to Doom-themes once it is polished.

**Comment by nrvale0** [comment]
Hey hey. I also have doom-themes via use-package though I didn't show that above.

So I went through this package and compared it to the loading process for themes that are shipped with the doom-themes package. I'm sure you know this but the doom-themes package has some utility functions that wrap things like `(deftheme ...)` etc so perhaps it's just as simple as some sort of pathing assumption about ~/.doom.

In any case, I think I saw elsewhere that you mentioned you were going to PR this against the upstream doom-themes package at some point. I'll just wait until it lands there as I use that package all the time with no problems. Looking forward to it. Particularly that dark theme. 

thx
